cnd/src/swap_protocols/swap_id.rs

```diff
@@ -5,6 +5,12 @@ use uuid::Uuid;
 #[derive(Serialize, Deserialize, Clone, Copy, Debug, Eq, Hash, PartialEq)]
 pub struct SwapId(pub Uuid);
 
+impl SwapId {
+    pub fn as_bytes(&self) -> &[u8] {
```

Comment:
This seems to be a relic from using byte arrays in the message.

Reply:
Nice catch, thanks.